### PR TITLE
Document `axtg` command

### PR DESCRIPTION
**Pull request by officialcjunior**
**Detailed description**

* Added a short description and an example for the `axtg` command
* Added information about some useful evars.

**Closing issues**

None

### CODE REVIEW

File: src/analysis/code_analysis.md [hunk 1]
```diff
@@ -194,6 +194,18 @@ sub.strlen_d50 0x5de0 [STRING] lea rcx, str.02x
 (nofunc) 0x17838 [CODE] jae str.02x
 ```
 
+There are also some useful commands under `axt`. You can use `axtg` to generate radare2 commands which you can use to create graphs according to the XREFs.
```
Comment (ITAYC0HEN) [inline src/analysis/code_analysis.md:197]:
```suggestion
There are also some useful commands under `axt`. Use `axtg` to generate radare2 commands which will help you to create graphs according to the XREFs.
```

File: src/analysis/code_analysis.md [hunk 2]
```diff
@@ -194,6 +194,18 @@ sub.strlen_d50 0x5de0 [STRING] lea rcx, str.02x
...
+
+```
+[0x08048320]> s main
+[0x080483e0]> axtg
+agn 0x8048337 "entry0 + 23"
+agn 0x80483e0 "main"
+age 0x8048337 0x80483e0
+```
+
+You can also run `axt*` to split the radare2 commands and set flags on those corresponding XREFs.
```
Comment (ITAYC0HEN) [inline src/analysis/code_analysis.md:207]:
```suggestion
Use `axt*` to split the radare2 commands and set flags on those corresponding XREFs.
```

File: src/configuration/evars.md [hunk 3]
```diff
@@ -23,7 +23,25 @@ Changes syntax flavor for disassembler between Intel and AT&T. At the moment, th
 
 ### asm.pseudo
 
-A boolean value to choose a string disassembly engine. "False" indicates a native one, defined by the current architecture, "true" activates a pseudocode strings format; for example, it will show `eax=ebx` instead of a `mov eax, ebx`.
+A boolean value to choose a string disassembly engine. "False" indicates a native one, defined by the current architecture, "true" activates a pseudocode strings format. For example, it'll transform :
```
Comment (ITAYC0HEN) [inline src/configuration/evars.md:26]: What is a "string disassembly engine". Isn't clear

Comment (officialcjunior) [reply src/configuration/evars.md:26]: I'm not sure too, it was already there.
How about:
`A boolean value to set the pseudo syntax in the disassembly` ?

Comment (ITAYC0HEN) [reply src/configuration/evars.md:26]: Much better :)
If you don't understand something, you would want to consider re-phrasing it in better words. Just as you did now <3

**Comment by officialcjunior** [comment]
Hey @Maijin 
Anything else to add here?

**Pull request merged**